Bottom: 087d0282c4f27cd4f751628cf9c8924dfbc2f91e
Top:    f7f32f0d0b345e2ce6b72b0896b3b949c281f30f
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   2010-11-26 04:04:41 +0000

Additional defines for a 3D joystick for (older) linux versions

Older versions of linux do not have these defines in <linux/input.h>

#define REL_RX  0x03
#define REL_RY  0x04
#define REL_RZ  0x05



---

diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index 6dd1b5f..618be32 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -21,6 +21,19 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
+/*
+Additional defines for a 3D joystick for (older) linux versions which do not
+have these defines in <linux/input.h>
+*/
+#ifndef REL_RX
+#define REL_RX                  0x03
+#endif
+#ifndef REL_RY
+#define REL_RY                  0x04
+#endif
+#ifndef REL_RZ
+#define REL_RZ                  0x05
+#endif
 
 #include <glib.h>
